tools/AzureMarkdownRewriterTool/Program.cs

```diff
@@ -386,8 +397,9 @@ private int Rewrite()
                             if (_isMigration)
                             {
                                 // Fixs Azure articles first in order to let docfx parse them correctly.
-                                source = FixAzureIncludeSyntax(source);
-                                source = FixAzureDefinitionSyntax(source);
+                                source = FixAzureIncludeWithPrefixSyntax(source);
+                                source = FixAzureIncludeWithPostfixSyntax(source);
+                                source = FixAzureDefinitionWithLeadingWhitespacesSyntax(source);
```

Comment:
minor: can we change this to a chaining call, like source.FixPrefix().FixPostFix().TrimWhiteSpaces(). I think it's cleaner